_data/nav.yml

```diff
@@ -22,6 +22,9 @@
     - subtitle: "Donate"
       subhref: "/donate.html"
 
+- title: "Call For Participation"
+  href: "/cfp.html"
+
```

Comment:
Adding to the NAV Bar. Happy to shorten the title to CFP if we start running out of menu space.

Reply:
Let's call it CFP now.

Reply:
Shortened to CFP